definitions/npm/ramda_v0.x.x/flow_v0.39.x-/ramda_v0.x.x.js

```diff
@@ -121,6 +129,7 @@ declare module ramda {
 
   declare var compose: Compose;
   declare var pipe: Pipe;
+  declare var pipeP: PipeP;
```

Comment:
I don't fully understand why they've chosen to use one style of declarations (intersection type plus `declare var`) for certain curried functions, like `pipe` and `pipeP`, and a different style of declarations (`declare function` variants) for other curried functions, like `tryCatch`.  Do you know?

It seems like the right thing to match the existing style, though, like you did :+1: